sprint0/sprint0/Bosses/Dodongo/Dodongo.cs

```diff
@@ -69,17 +69,28 @@ public Dodongo(Texture2D texture, Vector2 location, Game1 game) : base(texture,
 
         public override void Draw(SpriteBatch spriteBatch)
         {
-            if (direction == Direction.w || direction == Direction.e)
+            if (frameSpawn >= totalFramesSpawn * repeatedFramesSpawn)
             {
-                spriteBatch.Draw(Texture, Location, rightLeftSources[currentFrameRL / repeatedFrames],
-                    Color.White, 0, new Vector2(0, 0), spriteEffects[currentSpriteEffect / repeatedFrames], 0);
+                if (direction == Direction.w || direction == Direction.e)
+                {
+                    spriteBatch.Draw(Texture, Location, rightLeftSources[currentFrameRL / repeatedFrames],
+                        Color.White, 0, new Vector2(0, 0), spriteEffects[currentSpriteEffect / repeatedFrames], 0);
+                }
+                else
+                {
+                    spriteBatch.Draw(Texture, Location, upDownSources[currentFrameUD / repeatedFrames], Color.White,
+                        0, new Vector2(0, 0), spriteEffects[currentSpriteEffect / repeatedFrames], 0);
+                }
             }
             else
             {
-                spriteBatch.Draw(Texture, Location, upDownSources[currentFrameUD / repeatedFrames], Color.White,
-                    0, new Vector2(0, 0), spriteEffects[currentSpriteEffect / repeatedFrames], 0);
+                if (frameSpawn < totalFramesSpawn * repeatedFramesSpawn)
+                {
+                    spriteBatch.Draw(game.Content.Load<Texture2D>("Images/Link"), Location, sourcesSpawn[frameSpawn / repeatedFramesSpawn], Color.White);
```

Comment:
I would prefer for the cloud to be a separate sprite rather than included in a boss. see how you added to misc for other enemies